### ISSUE TITLE
DFD CMS language help

### ISSUE THREAD

**Issue by phae** [codeforamerica/content-team#72]
Slightly outside your usual domain, but I'd really like some help on the language used inside the CMS. There's some help-type text in it and various terms that humanise the git-based workflow that it would be good to work on. I'd like it to feel like a friendly and jovial, but professional, system.

LMK how this could work.

**Comment by danhon** [comment]
off the top of my head:

 - if you have screenshots, we could annotate
 - we could do it in person with print outs
 - we could set aside regular times each week to go through some
 - or just submit screens/flows as individual issues

How did you do stuff like this at GDS? I have a feeling you were all in a team that sat together, and er, just did it? (i.e. how, specifically, did you do it?)

cc @dschrimmer as an FYI

**Comment by phae** [comment]
Yes. Sat together at a computer and basically paired on content stuff, so
the content person learned a bit about how to change language in the
product and the nerd learned a bit about sounding like a human.
